lib/plugins/aws/deploy/lib/iam-policy-lambda-execution-template.json

```diff
@@ -1,5 +1,8 @@
 {
   "Type": "AWS::IAM::Policy",
+  "DependsOn": [
+    "IamRoleLambdaExecution"
```

Comment:
Hmm. Good question. It was merged here: https://github.com/nicka/serverless/commit/749204d9622f90d0d55655b040af7772f94c7765

I think that some piece of code will take care of this.